utils/land-patterns.stanza

```diff
@@ -1178,7 +1180,7 @@ public defn make-two-pin-chip-landpattern (length:Toleranced,
   make-two-pin-chip-landpattern(
     length, width, lead-length, lead-width?, 
     FABRICATION-TOLERANCE, PLACEMENT-TOLERANCE, 
-    density-level, polarized?,
+    density-level, polarized?, polarized?
```

Comment:
are you going to use, in the future, `make-two-pin-chip-landpattern` with different values for `anode-and-cathode?` and `polarized?`?

Reply:
Yes.